Dapper.Rainbow NET45/DatabaseAsync.cs

```diff
@@ -26,8 +23,8 @@ public partial class Table<T, TId>
                 paramNames.Remove("Id");
 
                 string cols = string.Join(",", paramNames);
-                string cols_params = string.Join(",", paramNames.Select(p => "@" + p));
-                var sql = "set nocount on insert " + TableName + " (" + cols + ") values (" + cols_params + ") select cast(scope_identity() as int)";
+                string colsParams = string.Join(",", paramNames.Select(p => "@" + p));

```

Comment:
again, this is just yak shaving; there's no reason to *reject* it, but ... I guess personally I'd only do this (creating work for the coder and the reviewer) if I was already making related changes to that area